src/os/bluestore/BlueFS.cc

```diff
@@ -2165,8 +2165,6 @@ BlueFS::FileWriter *BlueFS::_create_writer(FileRef f)
   for (unsigned i = 0; i < MAX_BDEV; ++i) {
     if (bdev[i]) {
       w->iocv[i] = new IOContext(cct, NULL);
```

Comment:
You may also want to move that initialization into FileWriter ctor.  And pass cct or corresponding lambda to create IOContext there.

Reply:
bdev[i] out of scope of FileWriter.